frontend/src/app/lib/inject.js

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
inject.js takes care of building the store and the app, and renders the component into the page once we are running in the browser. When running on the server, the return value is rendered to a static string.